src/augment-pr-entry/augment-pr-entry.js

```diff
@@ -74,16 +81,39 @@ export const addCreationDate = async (prNode, prData) => {
     prNumberAndTimestamp.appendChild(creationDateNode);
 };
 
+export const addUsernameWithLatestUpdate = async (prNode, prActivity) => {
+    // pull requests by default have the initial commit info as an activity
+    const mostRecentAction = prActivity.values[0];
+    let author = '';
+
+    // merges, commit updates
+    if (mostRecentAction.update) {
+        author = mostRecentAction.update.author.display_name;
+    } else if (mostRecentAction.approval) {
+        // approvals
+        author = mostRecentAction.approval.user.display_name;
+    } else if (mostRecentAction.comment) {
+        // comments
+        author = mostRecentAction.comment.user.display_name;
+    }
+
+    const prUpdateTime = prNode.querySelector('.pr-number-and-timestamp')
+        .firstElementChild;
+    prUpdateTime.append(` by ${author}`);
```

Comment:
This is cool! I would add a check though before appending the new DOM element just in case the API changes at some point

```js
    if (author) {
        const prUpdateTime = prNode.querySelector('.pr-number-and-timestamp')
            .firstElementChild;
        prUpdateTime.append(` by ${author}`);
    }
```

Reply:
Good catch.